lib/spack/spack/operating_systems/cnl.py

```diff
@@ -114,7 +114,8 @@ def make_compilers(self, compiler_id, paths):
             comp = cmp_cls(
                 spack.spec.CompilerSpec(name + '@' + v),
                 self, "any",
-                ['cc', 'CC', 'ftn'], [cmp_cls.PrgEnv, name + '/' + v])
+                ['cc', 'CC', 'ftn'], [cmp_cls.cray_prgenv_name,
+                                      name + '/' + v])
```

Comment:
Just put the *whole* array on the next line and line it up with the other parameters...